ntc_templates/templates/index

```diff
@@ -129,6 +129,7 @@ aruba_os_show_ip_interface_brief.textfsm, .*, aruba_os, sh[[ow]] ip in[[terface]
 aruba_os_show_ap_radio-database.textfsm, .*, aruba_os, show ap radio-database
 aruba_os_show_ap_database_long.textfsm, .*, aruba_os, show ap database long
 aruba_os_show_ap_database.textfsm, .*, aruba_os, show ap database
+aruba_os_show_vlan.textfsm, .*, aruba_os, sh[[ow]] vlan
```

Comment:
Based on the discussion here, I've adjusted the index regex to make `a` or `n` optional.
Thank you!

```suggestion
aruba_os_show_vlan.textfsm, .*, aruba_os, sh[[ow]] vl[[an]]
```